docs/guides/manage-identities/account-recovery.mdx

```diff
@@ -0,0 +1,187 @@
+import CodeBlock from '@theme/CodeBlock'
+import SelfHostedKratos from '@site/src/components/SelfHostedKratos'
+
+<Tabs
+  defaultValue="curl"
+  values={[
+    {label: 'curl', value: 'curl'},
+    {label: 'GoLang', value: 'go'},
+    {label: 'Self-Hosted Ory Kratos', value: 'kratos'},
+  ]}>
+  <TabItem value="curl">
+
+```shell
+curl --request POST -sL \
+  --header "Authorization: Bearer ory_pat_xRKLsFEOUFQFVBjd6o3FQDifaLYhabGd" \
+  --header "Content-Type: application/json" \
+  --request POST \
+  --data '{
+  "expires_in": "12h",
+  "identity_id": "e01b5f2f-6afc-4194-8578-4cebcf69a4d5"
+}' https://playground.projects.oryapis.com/api/kratos/admin/recovery/link
+```
+
+  </TabItem>
+  <TabItem value="go">
+
+```go
+package main
+
+import (
+	"fmt"
+	"github.com/ory/client-go/client"
+	"github.com/ory/client-go/client/admin"
+	"github.com/ory/client-go/models"
+)
+
+func main() {
+	c := client.New(nil, &client.TransportConfig{
+        Host: "https://playground.projects.oryapis.com",
+        BasePath: "/",
+        Schemes: []string{"http"},
+    })
+
+	res, err := c.Admin.CreateRecoveryLink(admin.NewCreateRecoveryLinkParams().WithBody(admin.CreateRecoveryLinkBody{
+		IdentityID: models.UUID("the-uuid"),
+	}))
+	if err != nil {
+		// ...
+	}
+
+	fmt.Printf("Use link: %s", *res.Payload.RecoveryLink)
+}
+```
+
+  </TabItem>
+  <TabItem value="kratos">
+    <SelfHostedKratos />
+  </TabItem>
+</Tabs>
+````
+
+The response contains a `recovery_link` value which is the link the user should
+use to set up his or her credentials (e.g. connect to a Social Sign In Provider,
+set up a password, ...):
+
+```json
+{
+  "recovery_link": "https://playground.projects.oryapis.com/api/kratos/public/self-service/recovery?flow=81c55cec-76fd-4907-bddf-cc112e835698&token=yM9nAZpPIjwccKh9qHRh8OfywZSRcr6q",
+  "expires_at": "2022-02-25T03:09:37.60684766Z"
+}
+```
+
+The user has only a limited amount of time to update their credentials once they
+use the recovery link. The time is the privileged session
+
+````mdx-code-block
+<Tabs
+  defaultValue="cloud"
+  values={[
+    {label: 'Ory Cloud', value: 'cloud'},
+    {label: 'Self-Hosted Ory Kratos', value: 'kratos'},
+  ]}>
+  <TabItem value="cloud">
+
+Configure the privileged session lifespan at [console.ory.sh/projects/current/session](https://console.ory.sh/projects/current/session).
+
+  </TabItem>
+  <TabItem value="kratos">
+
+```yml title="path/to/config.yml"
+selfservice:
+  flows:
+    settings:
+      privileged_session_max_age: 30m
+```
+
+  </TabItem>
+</Tabs>
+````
+
+If the user fails to set up his / her credentials in time, another recovery link
+needs to be issued and the user needs to re-do the flow.
+
+It is currently not possible to send the recovery link directly to a user's
+email, this feature is tracked as
+[#595](https://github.com/ory/kratos/issues/595).
+
+### Enable Account Recovery
+
+To enable recovery flows, make the following adjustments to your Ory Kratos
+configuration:
```

Comment:
```suggestion
To enable recovery flows, make the following adjustments to your Ory Kratos
configuration:
```

This will probably be covered in different documentation, but right now it is very unclear (from this document alone) how this is to be actually done in Ory Cloud.

Reply:
You're right, could you please add some screen shots to explain this a bit in a follow up PR?